indico/modules/events/abstracts/blueprint.py

```diff
@@ -70,3 +71,4 @@
 
 # Abstract-specific management
 _bp.add_url_rule('/manage/abstracts/<int:abstract_id>/', 'manage_abstract', RHManageAbstract)
+_bp.add_url_rule('/manage/abstracts/<int:abstract_id>/abstract.pdf', 'abstract_pdf_export', RHAbstractExportPDF)
```

Comment:
If this is the PDF export for the managers I would make that obvious both in the RH and the endpoint name.